core/src/main/java/org/nzbhydra/tasks/HydraTaskScheduler.java

```diff
@@ -99,14 +97,26 @@ private void scheduleTask(TaskRuntimeInformation runtimeInformation, boolean run
         if (runNow && !shutdownRequested) {
             scheduler.execute(runnable);
             scheduler.setRemoveOnCancelPolicy(true);
+            if (taskInterval == 0) {  //tasks with an interval of 0 aren't scheduled, so the taskInformation has to be updated here
+                taskInformations.put(task, new TaskInformation(task.name(), new Date().toInstant(), null));
+                return; //return here, don't schedule the task
+            }
+        }
+        if (!taskSchedules.containsKey(task.name())) { //On startup
```

Comment:
I had to move this down here, because when manually running a disabled task, it would otherwise enter this branch because disabled tasks aren't contained in `taskSchedules`.